lingua_franca/res/countries.json

```diff
@@ -0,0 +1,3227 @@
+[
+    {
+        "ISO3166-1-Alpha-3": "TWN",
+        "ISO3166-1-Alpha-2": "TW",
+        "official_name_fr": "",
+        "ISO4217-currency_name": "",
+        "ISO4217-currency_alphabetic_code": "",
+        "official_name_ar": "",
+        "official_name_es": "",
+        "official_name_cn": "",
+        "official_name_en": "",
+        "official_name_ru": "",
+        "Language": "zh-TW"
```

Comment:
should this be `BCP47-lang` to match the ISO spec'd params? I notice some are `ISO639-1` codes, so maybe including one or both and letting the parser decide what to return for `language` would make sense?

Reply:
this makes sense, the source data for this was noisy and wasnt always valid, i filtered results and only left langs that were 2 or 4 letter codes